lib/utils/utils.ts

```diff
@@ -214,3 +214,21 @@ export const isTransactionStuck = (timestamp: string): boolean => {
   const oneDayAgo = subDays(new Date(), 1);
   return isBefore(transactionDate, oneDayAgo);
 };
+
+// Convert base64url string to Uint8Array for WebAuthn API
+export const base64urlToUint8Array = (base64url: string): Uint8Array => {
```

Comment:
@LiorAgnin can this function move to backend? We previously had issues building app on native because of binary methods in frontend.